OpenNefia.Core/SaveGames/SaveGameSerializer.cs

```diff
@@ -45,6 +45,11 @@ public interface ISaveGameSerializer
         /// Loads the game state from the provided save.
         /// </summary>
         void LoadGame(ISaveGameHandle save);
+
+        /// <summary>
+        /// Resets all entites, allowing a new save to load.
```

Comment:
Nitpick: `entities`, and also it resets more state than just entities (maps, areas)

Reply:
changed in 2e3ad22